src/main/java/namor/template/calculator/ClassicCalculator.java

```diff
@@ -0,0 +1,70 @@
+package namor.template.calculator;
+
+import java.util.Arrays;
+import java.util.LinkedList;
+
+public class ClassicCalculator {
+
+    private LinkedList<String> operands;
+
+    public ClassicCalculator() {
+        this.operands = new LinkedList<>();
+    }
+
+    public double calculate(String arithmeticExpression) {
+
+        String[] operationElements = arithmeticExpression.split(" ");
+        operands.addAll(Arrays.asList(operationElements));
+
+        while(operands.size() > 1) {
+            doArithmeticOperation(operands.poll(), operands.poll(), operands.poll());
+        }
```

Comment:
So when you will change your List to local variable you will have two options:
- to add another argument to your `doArithmeticOperation()` method
      example: `doArithmeticOperation(operands, operands.poll(), operands.poll(), operands.poll());` 
- or to make `doArithmeticOperation()` not void but to return a result of operations
      example: `String result = doArithmeticOperation(operands, operands.poll(), operands.poll(), operands.poll();`
                      `operands.addFirst(result);`